.gitignore

```diff
@@ -66,6 +66,7 @@ firebase.json
 
 # build output
 build/
+dist/
```

Comment:
If this is added to .gitignore then all the files under it should be deleted before that. Otherwise we'll see changes to the files themselves as you can see below.